templates/admin/group/group_property_editor.html.ep

```diff
@@ -5,10 +5,10 @@
 >
     <div class="card-header">
         % if (is_admin) {
-            Edit job group properties
+            Edit group properties
```

Comment:
I am not yet ready to agree with this. Please see https://github.com/os-autoinst/openQA/pull/2536#discussion_r350833767 as well. I consider the single word "group" too ambiguous and "job group" a good generalization that can also cover parent job groups.

Reply:
Not sure what "group" is ambiguous with (within openQA) but I've changed it back.